Mathlib/Algebra/Ring/Subsemiring/Basic.lean

```diff
@@ -285,7 +296,6 @@ instance decidableMemCenter {R} [Semiring R] [DecidableEq R] [Fintype R] :
 theorem center_eq_top (R) [CommSemiring R] : center R = ⊤ :=
   SetLike.coe_injective (Set.center_eq_univ R)
 
-
```

Comment:
Is this intentional?

Reply:
Yes, it seems pointless to have two consecutive blank lines at the end of a section.